de.dlr.sc.overtarget.language.ui/src/de/dlr/sc/overtarget/language/ui/contentassist/UnitManager.xtend

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * Copyright (c) 2020 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.overtarget.language.ui.contentassist
+
+import java.util.HashMap
+import java.util.ArrayList
+import de.dlr.sc.overtarget.language.targetmodel.Unit
+import de.dlr.sc.overtarget.language.targetmodel.TargetModel
+import org.eclipse.ui.PlatformUI
+import org.eclipse.xtext.ui.editor.XtextEditor
+import org.eclipse.core.runtime.jobs.Job
+import org.eclipse.core.runtime.IStatus
+import org.eclipse.core.runtime.IProgressMonitor
+import de.dlr.sc.overtarget.language.util.QueryManager
+import org.eclipse.core.runtime.Status
+import org.eclipse.core.runtime.CoreException
+import java.io.IOException
+import de.dlr.sc.overtarget.language.Activator
+import org.eclipse.ui.statushandlers.StatusManager
+import de.dlr.sc.overtarget.language.targetmodel.TargetFile
+
+class UnitManager {
+	
+	static HashMap<String, ArrayList<Unit>> mapOfUnits = new HashMap<String, ArrayList<Unit>>
```

Comment:
I think having this class as [Singelton](https://de.wikibooks.org/wiki/Muster:_Java:_Singleton) would be cleaner than a static map here?